src/encoding.md

```diff
@@ -344,6 +345,60 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-invalid-csv]: #ex-invalid-csv
+<a name="ex-invalid-csv"></a>
+## Handle invalid CSV data with Serde
+
+[![csv-badge]][csv] [![serde-badge]][serde] [![cat-encoding-badge]][cat-encoding]
+
+We get deserialization errors if [`invalid CSV data`] is encountered. [`Serde`] is
```

Comment:
Please remove the backticks from "invalid CSV data" - backticks are for formatting text as code and this is just regular text. Same with "Serde".